src/router/asyncComponentImport.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
```

Comment:
Let's drop this for now. 
We can implement this later while code splitting.

Reply:
done